app/registration/views/addressFields.html

```diff
@@ -14,7 +14,7 @@
                    bahmni-autocomplete source="getAddressEntryList(addressLevel.addressField)"
                    response-map="getAddressDataResults" on-select="addressFieldSelected(addressLevel.addressField)"
                    ng-change="clearFields(addressLevel.addressField)"
-                   class="form-control keyboard-in">
+                   class="form-control keyboard-in" ng-pattern="/^[a-zA-Z\s]*$/">
```

Comment:
I think this regex should be modified to include numbers and some other characters. If it is left to be the way it is, then things like **Distrito Municipal de KaMubukwane (antigo Distrito Urbano N 5)**  will not be allowed for **Distro**
![distro](https://cloud.githubusercontent.com/assets/1763308/22098351/dea3ea04-de36-11e6-9bb1-3bbb7dab5294.png)

Or else those items need to be renamed to match the supplied rejex